src/locales/en.js

```diff
@@ -24,6 +24,7 @@ export default {
   'logout.button': 'LOGOUT',
 
   'home.welcome': 'Welcome to React Redux Base {email}',
+  'home.current-location': 'You are here!',
```

Comment:
```suggestion
  'home.current_location': 'You are here!',
```
or camelcase but should be consistent with the other strings